### DISCUSSION TITLE
Remove property from existing schema

### DISCUSSION THREAD

**Discussion by davideroffo** [sinclairzx81/typebox#589] [Q&A] [answered]
Hello,
I'm trying to remove a property from an existing schema, and I want to save this updated schema (without the specified property) into another schema. 
Let me give you an example:

```javascript
const UserSchema1 = Type.Strict(
  Type.Object({
    userId: Type.String({ format: 'uuid' }),
    password: Type.String(),
  }),
)

const UserSchema2 =  Type.Omit(UserSchema1, ['password'])
```

But if I `console.log` the UserSchema2 properties I get:

```javascript
console.log(UserSchema2.properties)

{
  userId: { format: 'uuid', type: 'string' },
  password: { type: 'string' }
}
```

Am I missing something?

Thank you,

**Comment by sinclairzx81** [comment, marked as answer]
@davideroffo Hi,

Try removing Type.Strict from the outer schema. The Strict call actually removes compositing symbols that TypeBox needs to remap types. In most cases, you would only call strict on a schema if you are sure you've finished remapping the types, but more often than not, it should be fine to retain the symbols (as they do not serialize to Json are are compatible with Ajv and other validators)

Hope this helps
